.buildkite/pipeline.yml

```diff
@@ -697,3 +697,57 @@ steps:
     trigger: "beats-agent-serverless-tests"
     build:
       branch: "main"
+
+  - label: "Trigger DRA (SNAPSHOT)"
```

Comment:
Might be beneficial to say `Trigger DRA (SNAPSHOT) in dry-run mode" or something similar